server/src/main/java/org/opensearch/action/admin/cluster/health/ClusterHealthRequest.java

```diff
@@ -55,6 +58,7 @@
 public class ClusterHealthRequest extends ClusterManagerNodeReadRequest<ClusterHealthRequest> implements IndicesRequest.Replaceable {
 
     private String[] indices;
+    private String awarenessAttribute = "";
```

Comment:
why empty?

Reply:
Setting a default value for serialization.